src/https.js

```diff
@@ -17,7 +17,10 @@ async function get(uri) {
         if (res.statusCode !== 200) {
           return reject(new Error(results.toString()));
         } else {
-          return resolve(JSON.parse(results));
+          const resultsParsed = JSON.parse(results);
```

Comment:
I'm thinking it would be cleaner to have an options argument passed to `http.get` that signals you want a result object instead of the parsed result. The result object should include a `result` property with the parsed result and a `headers` property that includes the response headers. This way we are not taking the hit of a full properties copy and attaching a non-sematntic hidden property to it.

I  would also be ok making a second method `https.getResult` that has the same logic as `http.get` but returns a result object as described above.